core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -980,7 +1004,7 @@ public static Object percent3( ValueMetaInterface metaA, Object dataA, ValueMeta
       case ValueMetaInterface.TYPE_BIGNUMBER:
         return metaA.getBigNumber( dataA ).add(
           divideBigDecimals( multiplyBigDecimals(
-            metaB.getBigNumber( dataB ), metaA.getBigNumber( dataA ), null ), new BigDecimal( 100 ), null ) );
+            metaB.getBigNumber( dataB ), metaA.getBigNumber( dataA ), null ), new BigDecimal( 100 ) ) );
```

Comment:
Need to put this back - cannot change the API

Reply:
Done